app/nasa_collaboration/page.module.css

```diff
@@ -1,12 +1,66 @@
-.app {
-  background-color: black;
+ .app {
   width: 100vw;
   height: 100vh;
 }
+  .astronomyContainer {
```

Comment:
In CSS, the convention is to use [kebab case](https://developer.mozilla.org/en-US/docs/Glossary/Kebab_case) for class names :)

Reply:
Thanks, I did not know that, I will change it.